app/javascript/components/public_health/PatientsTable.js

```diff
@@ -475,7 +476,7 @@ class PatientsTable extends React.Component {
   };
 
   formatTimestamp(data) {
-    const timestamp = data.value;
+    const timestamp = Object.prototype.hasOwnProperty.call(data, 'value') ? data.value : data;
```

Comment:
is this just to make sure we're not calling `.value` on `data` if `data` is `undefined` or `null`? if so, I think we can just use the safe navigation operator `data?.value`

Reply:
With [Dave's PR](https://github.com/SaraAlert/SaraAlert/pull/741), this method will actually be removed and a more generally available [timestamp format helder function](https://github.com/SaraAlert/SaraAlert/pull/741/files#diff-31ba3ada190c965136dfcc817ee8ab65f55dc7a97773a7d858978ece4f531fb6R7) will be used in this file instead. So I just copied and pasted this relevant line from Dave's PR since this is what it will end up being after it's merged which should be soon. But if you still think it's worth making that adjustment, let me know and I can certainly do so.

Reply:
Yeah I saw that Becky. I think it's okay to leave this as is. It's a good confirmation that my PR isn't going to change any of the behavior of yours (since the logic is the same).